models/gc_nnunet_pancreas/utils/cli.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
Why again did we have the CLI here? 

It'd be beneficial if you could add 1-2 lines of explanation to the file's header why we chose to wrap our code in a cli, what problems that solved etc. to make it easier for future submissions to learn from existing ones.

Reply:
The model uses an nn-UNet internally, so it would produce uncaptured stdout output otherwise. I'll add a few lines of explanation to the file's header.

Reply:
Done